components/kyma-environment-broker/internal/process/provisioning/external_eval.go

```diff
@@ -24,6 +24,7 @@ func NewExternalEvalCreator(delegator *avs.Delegator, disabled bool, assistant *
 
 func (eec *ExternalEvalCreator) createEval(operation internal.ProvisioningOperation, url string, logger logrus.FieldLogger) (internal.ProvisioningOperation, time.Duration, error) {
 	if eec.disabled {
```

Comment:
do we need this check at all?

Reply:
imo we should, because this is set by this flag globally:
https://github.com/kyma-project/control-plane/blob/81c77a1ef304437c78d877c740bf814e68c34804/components/kyma-environment-broker/internal/avs/config.go#L17

and we use it on `stage`